tests/lib/rules/jsx-one-expression-per-line.js

```diff
@@ -937,26 +937,6 @@ foo
         <App>
           <Foo></
         Foo>
-</App>
-      `,
-      errors: [
-        {
-          messageId: 'moveToNewLine',
-          data: { descriptor: 'Foo' },
-        },
-      ],
-      parserOptions,
-    },
-    {
-      code: `
-        <App>
-          <Foo></
-        Foo></App>
-      `,
-      output: `
-        <App>
-          <Foo></
-        Foo>
```

Comment:
Again two same in a row:

https://github.com/jsx-eslint/eslint-plugin-react/blob/8e1a94b67d081fdc132e9a7e175db3fbf2e02956/tests/lib/rules/jsx-one-expression-per-line.js#L930-L969